### ISSUE TITLE
TextCondition getSelectedOptionsTexts() not working on disable elements with Selenium 4.5.0

### ISSUE THREAD

**Issue by evgeny-tumashchick** [selenide/selenide#2018]
## The problem
The problem actually connected with update to Selenium 4.5.0 in Selenide 6.9.0 version.
Selenium changed verification for org.openqa.selenium.support.ui.Select#Select constructor and now require element to be enabled on UI.

Actually now it causes failures for verification where the following condition was used for such elements:
`$`(locator).shouldHave(**Condition.text(myText)**)

The method which cause the issues:
com.codeborne.selenide.conditions.TextCondition#getSelectedOptionsTexts
```
  protected String getSelectedOptionsTexts(WebElement element) {

// Here new Select element is created
    List<WebElement> selectedOptions = new Select(element).getAllSelectedOptions();
    return selectedOptions.stream().map(WebElement::getText).collect(joining());
  }
```

## Details
See the following diff in Select constructor:

**How it was in selenium 4.4.0**
org.openqa.selenium.support.ui.Select#Select

```
  public Select(WebElement element) {
    String tagName = element.getTagName();

    if (null == tagName || !"select".equals(tagName.toLowerCase())) {
      throw new UnexpectedTagNameException("select", tagName);
    }

    this.element = element;

    String value = element.getDomAttribute("multiple");

    // The atoms normalize the returned value, but check for "false"
    isMulti = (value != null && !"false".equals(value));
  }

```
**How it is now in selenium 4.5.0**
org.openqa.selenium.support.ui.Select#Select

```
  public Select(WebElement element) {
    String tagName = element.getTagName();

    if (!"select".equalsIgnoreCase(tagName)) {
      throw new UnexpectedTagNameException("select", tagName);
    }

   // This new part of code here causes the issue

    if (!element.isEnabled()) {
      throw new UnsupportedOperationException("Select element is disabled and may not be used.");
    }

    this.element = element;

    String value = element.getDomAttribute("multiple");

    // The atoms normalize the returned value, but check for "false"
    isMulti = (value != null && !"false".equals(value));
  }
```

## Tell us about your environment
* **Selenide Version**: 6.9.0
* **Chrome\Firefox\IE Version**: 105.0.5195.52
* **Browser Driver Version**: 
* **Selenium Version**: 4.5.0
* **OS Version**: 13.0 (22A380)

## Code To Reproduce Issue [ Good To Have ]
Described the conditions in Details section

**Comment by asolntsev** [comment]
@evgeny-tumashchick After a long discussion, this problem was fixed in Selenium 4.6.0.
Even more, in Selenide 6.10.0 the work with `<select>` elements will be re-implemented using JavaScript code, see PR https://github.com/selenide/selenide/pull/1876

**Comment by evgeny-tumashchick** [comment]
@asolntsev thank you for detailed response. Will wait for Selenide 6.10.0 release
